src/components/atoms/SubTab/SubTab.tsx

```diff
@@ -0,0 +1,18 @@
+import clsx from 'clsx';
+import { SubTabProps } from './types';
+
+export const SubTab = ({ text, active }: SubTabProps) => {
+    return (
+        <span
+            className={clsx(
+                'group flex h-full w-full items-center justify-center rounded-md px-[5.88px] py-1.5 text-xs font-semibold leading-4 duration-300 hover:opacity-100 hover:ease-in-out laptop:px-3 laptop:leading-5',
```

Comment:
please use the respective typography

Reply:
@as472 we don't have typography for mobile in figma: https://www.figma.com/design/YIBsk1ihFbPlDb8XldGj1T/SF-Design-System?node-id=614-6853&m=dev

so I'll use the tailwind classes for mobile, then do a laptop:typography-caption-2 with it (for desktop)

Reply:
use the new classes to define the typography.
typography-desktop-body-5